ts/Series/HollowCandlestick/HollowCandlestickSeries.ts

```diff
@@ -169,31 +169,26 @@ class HollowCandlestickSeries extends CandlestickSeries {
      */
     public getPriceMovement(): void {
         const series = this,
-            // procesed and grouped data
+            // Processed and grouped data
             processedYData = series.allGroupedData || series.yData,
             hollowCandlestickData = this.hollowCandlestickData;
 
-        if (
-            !hollowCandlestickData.length &&
-            processedYData &&
-            processedYData.length
-        ) {
-
-            // First point is allways bullish (transparent).
-            hollowCandlestickData.push({
-                isBullish: true,
-                trendDirection: 'up'
-            });
-
-            for (let i = 1; i < processedYData.length; i++) {
-                const dataPoint: any = processedYData[i],
-                    previousDataPoint: any = processedYData[i - 1];
-
-                hollowCandlestickData.push(series.isBullish(
-                    dataPoint,
-                    previousDataPoint
-                ));
-            }
+        hollowCandlestickData.length = 0;
```

Comment:
Why do we need to clear this always? I think we have cleared this in `updatedData` event 🤔 If not, perhaps we should add `updated` event too? Updating this on each chart's redraw seems a bit too aggressive.

Reply:
Previously, it didn't work at all. Similar scenario to heikinashi, `hollowCandlestickData` had a wrong length.
Not working on master: https://jsfiddle.net/BlackLabel/6um2gctx/

Reply:
Yes, I know, I know. Just wanted to clarify if we need to clear that array length on each redraw. It seems heikinashi has the same problem. We clear and build the dataset on each redraw (aaand we have event on updated data which is redundant now, isn't it?). Let's keep this solution for now, however adding this to the Performance profile

Reply:
Sounds fair! I'll add the test, and we are good to go. We can discuss it later F2F, since it's a good case to work on perf optimization.